### PR TITLE
New package: swarp-0.1

### PR DESCRIPTION
**Pull request by ibabushkin**
The packaged software is rather trivial, but quite useful at times - so it makes sense to include it in the repos IMO.

### CODE REVIEW

File: srcpkgs/swarp/template [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+version=0.1
+revision=1
+short_desc="Simple pointer warping tool for X"
+maintainer="Inokentiy Babushkin <inokentiy.babushkin@googlemail.com>"
+license="MIT"
+makedepends="libX11-devel"
+homepage="http://tools.suckless.org/x/swarp"
+distfiles="http://dl.suckless.org/tools/${pkgname}-${version}.tar.gz"
+checksum=ef5730fe8ee00879cbec1e91e22a7f0f7817a63375d790d775f5b7427886d45f
+
```
Comment (ebfe) [inline srcpkgs/swarp/template]: judging from the sed's already done, it doesn't look like that would improve the template.

Comment (SirDoctors) [inline srcpkgs/swarp/template]: If thats the case then which would be a the preferred way then? To remove the sed's or to set the build style?

Comment (ebfe) [inline srcpkgs/swarp/template]: neither would work, so i'd just leave it as it is.

Comment (ibabushkin) [inline srcpkgs/swarp/template]: This is based on other suckless software packaged for void - for instance `slock`. Those packages use the same approach.

**Pull request merged**